src/IFrameNavigator.ts

```diff
@@ -102,10 +103,11 @@ export default class IFrameNavigator implements Navigator {
     private annotator: Annotator | null;
     private settings: BookSettings;
     private eventHandler: EventHandler;
+    private upUrl: URL | null;
+    private upLabel: string | null;
```

Comment:
I think I might come back to this later and make this constructor take one config object as its argument.